sensor_msgs/PointCloud2 pc2

---

bool success   # indicate successful run of triggered service
string message # informational, e.g. for error messages